A3-Algae-Spec/Algae.Rmd

```diff
@@ -23,15 +23,14 @@ Be able to...
   
 ## 1. Getting Started
 
-### 1a. Put data in long form on Google Sheets
+### 1a. Put data in long form on Google Sheets and adjust share it with your team
 
-Put your group's data into a Google Sheet in _*long form*_. Set it up with the column names `Algae` (1,2,3), `Wavelength` (400 to 700, by tens), and `Absorption` (what you measured). Remember that there can only be one datapoint per line. You should have 1 header row and 93 rows of data total.
+Put your group's data into a Google Sheet in _*long form*_. Set it up with the column names `Algae` (1,2,3), `Wavelength` (400 to 700, by tens), and `Absorption` (what you measured). You should have 1 header row and 93 rows of data total.
```

Comment:
I really want to include "Remember that there can only be one datapoint per line. " - this is the key to long form data

Reply:
fixed next commit